### ISSUE TITLE
main url crash

### ISSUE THREAD

**Issue by teekcode** [Codewars/codewars.com#1429]
visiting https://www.codewars.com/ will crash chrome's tab( 66.0.3359.170), happen in firefox also.
![screenshot_20180512231329](https://user-images.githubusercontent.com/6964005/39958765-1c0496d8-563a-11e8-8383-7757a1f62bfa.png)

**Comment by kazk** [comment]
Are you using Windows? See #1157

**Comment by teekcode** [comment]
@kazk yes, and it is only happen in https://www.codewars.com/, if I try to access https://www.codewars.com/users/sign_in, it is ok.
